docs/guide/en/field-configuration.md

```diff
@@ -1,33 +1,150 @@
-# Fields configuration
+# Field configuration
 
-Fields could be configured on multiple levels:
+Fields could be configured on multiple levels.
```

Comment:
Need to add form specific content and link https://github.com/yiisoft/widget/pull/107